src/main/java/org/gridsuite/modification/server/entities/equipment/creation/StaticCompensatorCreationEntity.java

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.entities.equipment.creation;
+
+import com.powsybl.iidm.network.StaticVarCompensator;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.gridsuite.modification.server.dto.ModificationInfos;
+import org.gridsuite.modification.server.dto.StaticVarCompensatorCreationInfos;
+import org.gridsuite.modification.server.dto.VoltageRegulationType;
+import org.gridsuite.modification.server.entities.equipment.modification.FreePropertyEntity;
+import org.springframework.util.CollectionUtils;
+
+/**
+ * @author Ghazwa Rehili <ghazwa.rehili at rte-france.com>
+ */
+@NoArgsConstructor
+@Getter
+@Entity
+@Table(name = "staticVarCompensatorCreation")
+@PrimaryKeyJoinColumn(foreignKey = @ForeignKey(name = "staticVarCompensatorCreation_id_fk_constraint"))
+public class StaticCompensatorCreationEntity extends InjectionCreationEntity {
+    @Column
+    private Double maxSusceptance;
+
+    @Column
+    private Double minSusceptance;
+
+    @Column
+    private Double maxQAtNominalV;
+
+    @Column
+    private Double minQAtNominalV;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "regulationMode")
+    private StaticVarCompensator.RegulationMode regulationMode;
+
+    @Column
+    private Double voltageSetpoint;
+
+    @Column
+    private Double reactivePowerSetpoint;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "voltageRegulationType")
+    private VoltageRegulationType voltageRegulationType;
+
+    @Column(name = "regulatingTerminalId")
+    private String regulatingTerminalId;
+
+    @Column(name = "regulatingTerminalType")
+    private String regulatingTerminalType;
```

Comment:
same here

Reply:
I used the same to be coherent with GeneratorCreation and TapChangerCreation